test/easy_localization_plural_rules_unit_test.dart

```diff
@@ -0,0 +1,212 @@
+    });
+  });
+
+  group("Arabic plural", () {
+    test('Arabic few (no ignorePluralRules)', () async {
+      Localization.load(
+        const Locale('ar'),
+        translations: r.translations,
+        fallbackTranslations: r.fallbackTranslations,
+        ignorePluralRules: false,
+      );
+      expect(Localization.instance.plural('hat', 2), 'two hats');
+      expect(Localization.instance.plural('hat', 10), 'few hats');
+      expect(Localization.instance.plural('hat', 110), 'few hats');
+    });
+    test('Arabic few (with ignorePluralRules)', () async {
+      Localization.load(
+        const Locale('ar'),
+        translations: r.translations,
+        fallbackTranslations: r.fallbackTranslations,
+        ignorePluralRules: true,
+      );
+      expect(Localization.instance.plural('hat', 2), 'two hats');
+      expect(Localization.instance.plural('hat', 3), 'other hats');
+      expect(Localization.instance.plural('hat', 10), 'other hats');
+      expect(Localization.instance.plural('hat', 110), 'other hats');
+    });
+    test('Arabic many (no ignorePluralRules)', () async {
+      Localization.load(
+        const Locale('ar'),
+        translations: r.translations,
+        fallbackTranslations: r.fallbackTranslations,
+        ignorePluralRules: false,
+      );
+
+      expect(Localization.instance.plural('hat', 11), 'many hats');
+      expect(Localization.instance.plural('hat', 111), 'many hats');
+    });
+    test('Arabic many (with ignorePluralRules)', () async {
+      Localization.load(
+        const Locale('ar'),
+        translations: r.translations,
+        fallbackTranslations: r.fallbackTranslations,
+        ignorePluralRules: true,
+      );
+      expect(Localization.instance.plural('hat', 10), 'other hats');
+      expect(Localization.instance.plural('hat', 11), 'other hats');
+      expect(Localization.instance.plural('hat', 111), 'other hats');
+    });
+    test('Arabic all cases (no ignorePluralRules)', () async {
+      Localization.load(
+        const Locale('ar'),
+        translations: r.translations,
+        fallbackTranslations: r.fallbackTranslations,
+        ignorePluralRules: false,
+      );
+
+      expect(Localization.instance.plural('hat', 0), 'no hats');
+      expect(Localization.instance.plural('hat', 1), 'one hat');
+      expect(Localization.instance.plural('hat', 2), 'two hats');
+      expect(Localization.instance.plural('hat', 3), 'few hats');
+      expect(Localization.instance.plural('hat', 4), 'few hats');
+      expect(Localization.instance.plural('hat', 11), 'many hats');
+      expect(Localization.instance.plural('hat', 101), 'other hats');
+      expect(Localization.instance.plural('hat', 111), 'many hats');
+      expect(Localization.instance.plural('hat', 103), 'few hats');
+    });
+    test('Arabic all cases (with ignorePluralRules) | using `_pluralCaseFallback`', () async {
+      Localization.load(
+        const Locale('en'),
```

Comment:
Ar?

Reply:
Great note. It has been resolved. Please check it now :)